### PR TITLE
Stub the y2r:u service

### PR DESCRIPTION
**Pull request by archshift**

### CODE REVIEW

**Comment by Subv** [comment]
LGTM

**Comment by purpasmart96** [comment]
LGTM

File: src/core/hle/service/y2r_u.h [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+// Namespace Y2R_U
+
+namespace Y2R_U {
+
+class Interface : public Service::Interface {
+public:
+    Interface();
+
+    std::string GetPortName() const override {
+    return "y2r:u";
```
Comment (Subv) [inline src/core/hle/service/y2r_u.h]: Indentation here

Comment (purpasmart96) [inline src/core/hle/service/y2r_u.h]: @Subv Looks fine to me.

**Comment by yuriks** [comment]
Does this help boot anything?

**Comment by Subv** [comment]
Gets Aero Porter a bit further, needs one of the functions in here to get even further

**Comment by archshift** [comment]
Fixed the indentation.

**Comment by Subv** [comment]
LGTM

**Comment by bunnei** [comment]
@yuriks it shouldn't help boot anything, no functions are implemented.
LGTM.

**Pull request merged**